cfme/common/provider_views.py

```diff
@@ -547,19 +541,21 @@ class ProviderEditView(ProviderAddView):
     @property
     def is_displayed(self):
         provider_obj = self.context['object']
-        expected_title = ("Edit {type} Providers '{name}'".format(
-            type=provider_obj.string_name, name=provider_obj.name))
+        expected_title = "Edit {type} Providers '{name}'".format(type=provider_obj.string_name,
+                                                                 name=provider_obj.name)
 
-        return (self.logged_in_as_current_user and
-                self.title.text == expected_title and
-                self.navigation.currently_selected == [
-                    'Compute',
-                    ('Clouds' if provider_obj.string_name == "Cloud" else provider_obj.string_name),
-                    'Providers'
-                ])
+        return (
+            self.logged_in_as_current_user and
+            self.title.text == expected_title and
+            self.navigation.currently_selected == [
+                'Compute',
+                ('Clouds' if provider_obj.string_name == "Cloud" else provider_obj.string_name),
+                'Providers'
+            ]
+        )
 
 
-class ContainerProviderEditViewUpdated(ProviderEditView, ContainerProviderSettingView):
+class ContainerProviderEditView(ProviderEditView, ContainerProviderSettingView):
     """
      Additional widgets for builds 5.9 and up
     """
```

Comment:
Need to update this doc string

Reply:
I don't think so, its accurate.  Don't need to eliminate occurances of 59 strings, just logic using that version.  For docblocks like this, I'd like to keep statements like this indicating when something started to be supported.